apps/console/src/public/deployment.config.json

```diff
@@ -348,6 +348,7 @@
                 "disabledFeatures": [
                     "applications.loginFlow.legacyEditor",
                     "applications.loginFlow.ai",
+                    "applications.myaccount.simplifiedSettings",
```

Comment:
We are only adding an overview page for myaccount app in this PR, aren't we? in that case, wouldn't it be better to rename this to something like `applications.myaccount.saasMyAccountOverview`?

Reply:
Not only that, we are also removing general, protocol and info tabs as well